x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/index/store/BaseSearchableSnapshotIndexInput.java

```diff
@@ -149,29 +122,4 @@ protected final boolean assertCurrentThreadMayAccessBlobStore() {
         return true;
     }
 
-    private long getPartNumberForPosition(long position) {
```

Comment:
Pushed these methods down to the cached index input only, the other implementation doesn't use them.